### DISCUSSION TITLE
Does LazyVim pin its plugins?

### DISCUSSION THREAD

**Discussion by Cretezy** [LazyVim/LazyVim#1009] [General]
Hi, I'm working on a content piece which talks about LazyVim, and I was wonder:

Does LazyVim pin plugins?

From that, I mean, when updating LazyVim, are any of its dependencies pinned inside LazyVim (vs just in lazy.nvim). 

Thank you!

**Comment by amaanq** [comment]
lazy.nvim and LazyVim itself are pinned (to stable releases), the rest are not - see https://www.lazyvim.org/configuration/lazy.nvim

**Comment by folke** [comment]
Nope, LazyVim is always updating plugins to the latest version.

I've tried using stable releases only before, but that lead to a lot of issues.

Always using the latest version is strangely enough much more stable.

**Reply by Cretezy** [reply]
Thank you for the reply!
